### DISCUSSION TITLE
USB drives could be a problem

### DISCUSSION THREAD

**Discussion by justinclift** [Romaq/bigrig-scripts#3] [Q&A] [answered]
> ... and a USB bay of five 3.64 TiB SATA drives.

Is there *any* way of connecting the drives other than USB?

USB storage has a terrible reputation for randomly dropping drives out of an array when they're used in anger (aka passing a lot of storage traffic), especially when they get hot. :worried:

**Comment by Romaq** [comment, marked as answer]
There is, but only *if* the client who directed me to set this up was also willing to pay for hardware with a drive controller and room to put the drives in.

https://store.minisforum.com/collections/all-product/products/minisforum-um790-pro?utm_source=bing&utm_medium=cpc&utm_campaign=S-UM790Pro-0523-EN&utm_term=um790%20pro&utm_content=广告组%201

This is what he's using for his small office, so that's what he sent me to use for testing and development of the hardware he wants to move his current work to. I do understand and agree with your concern, quite valid in an Enterprise environment. But as I'm not the fellow cutting the checks, and I was directed to base this on the use case of my "SOHO" environment, I'm limited to what I can connect on the butt-end of the hardware linked above with the SATA "duplication/clone bay" provided... that connects via USB.

Would I recommend the purchase of this for a business? Absolutely not, for the reasons that concern you. Would I suggest it for someone with "some" money for a home LAN that just wants to dink around? Yeah, so long as they understand the *liability* given the cost. You DO get what you pay for.

Beyond that, I agree with you, but it's my opportunity rather than my choice.

**Reply by justinclift** [reply]
No worries.  Who knows, maybe the 40Gb USB advertised on the packaging will turn out to be super reliable or something.  Hopefully that's the direction it goes. :smile:
